CMakeLists.txt

```diff
@@ -131,9 +132,11 @@ endif()
 find_package(LIBSSH 0.7.1)
 
 set(CMAKE_FIND_PACKAGE_PREFER_CONFIG ON)
-set(protobuf_MODULE_COMPATIBLE ON CACHE BOOL "" FORCE)
-find_package(Protobuf REQUIRED)
-find_program(PROTOC_COMPILER protoc REQUIRED)
+if(WITH_PROTOBUF)
+    set(protobuf_MODULE_COMPATIBLE ON CACHE BOOL "" FORCE)
+    find_package(Protobuf REQUIRED)
+    find_program(PROTOC_COMPILER protoc REQUIRED)
+endif()
```

Comment:
Is the `NO_PROTOBUF` define done somehow automatically by cmake?  I'm not seeing anything in the cmake code that defines it?

Reply:
It's defined in wpimath/CMakeLists.txt.

Reply:
I think it would be better to define in wpiutil, since that's where Protobuf.h is?  That way it will propagate everywhere.